state-chain/pallets/cf-elections/src/electoral_systems/blockchain/delta_based_ingress.rs

```diff
@@ -267,28 +326,20 @@ where
 			}
 
 			let mut election_access = electoral_access.election_mut(election_identifier)?;
-			if !closed_channels.is_empty() {
-				for closed_channel in closed_channels {
-					pending_ingress_totals.remove(&closed_channel);
-					channels.remove(&closed_channel);
-				}
 
-				if channels.is_empty() {
-					election_access.delete();
-				} else {
-					election_access.set_state(pending_ingress_totals)?;
-					election_access.refresh(
-						// This value is meaningless. We increment as it is required to use a new
-						// higher value to refresh the election.
-						election_identifier
-							.extra()
-							.checked_add(1)
-							.ok_or_else(CorruptStorageError::new)?,
-						channels,
-					)?;
-				}
+			for closed_channel in closed_channels {
+				pending_ingress_totals.remove(&closed_channel);
+				channels.remove(&closed_channel);
+			}
+
+			if channels.is_empty() {
+				election_access.delete();
 			} else {
-				election_access.set_state(pending_ingress_totals)?;
```

Comment:
> `election_access.set_state(pending_ingress_totals)?;`

Am I right that this is no longer needed at all? And if we never call set_state, do we need to read the `pending_ingress_totals` from state at all?